cogboard-webapp/src/components/widgets/dialogFields/SonarQubeMetricsInput.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import styled from '@emotion/styled/macro';
+
+import Input from '@material-ui/core/Input';
```

Comment:
I prefer named imports, like { Input } from '@material-ui/core'. It is more succinct.